RiotSharpTest/TournamentRiotApiTest.cs

```diff
@@ -74,26 +71,19 @@ public void GetTournamentMatch_Test()
         [TestCategory("TournamentRiotApi"), TestCategory("Async")]
         public void CreateProviderAsync_CreateTournamentAsync_CreateTournamentCodeAsync_UpdateTournamentCodeAsync_Test()
```

Comment:
Very weird indeed, could we ignore it and mention that it's not failing if ran alone?